ansible/roles/kubernetes-cni/defaults/main.yml

```diff
@@ -4,3 +4,5 @@ kubernetes_cni:
   calico:
     manifest_url: "https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml"
     calicoctl_url: "https://github.com/projectcalico/calicoctl/releases/download/v1.6.1/calicoctl"
+  flannel:
+    manifest_url: "https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml"
```

Comment:
This should most likely pull from a specific tag instead of master, such as: https://raw.githubusercontent.com/coreos/flannel/v0.10.0/Documentation/kube-flannel.yml